tests/integration/telemetry/stackdriver/common.go

```diff
@@ -359,10 +359,9 @@ func logDiff(t test.Failer, tp string, query map[string]string, entries []map[st
 	for _, s := range entries {
 		b, _ := json.Marshal(s)
 		ss := string(b)
-		if seen.Contains(ss) {
+		if seen.InsertContains(ss) {
```

Comment:
nit: this is why i was trying to brainstorm naming -- here the code is clearer (imho) as written previously; even tho the collection is not modified in the new example, that takes a second to grok with the new name (in this particular setting).